src/resources/Availability.js

```diff
@@ -0,0 +1,21 @@
+import Resource from './Resource'
+
+export default class Availability extends Resource {
+  source = 'availability';
+
+  async list ({
```

Comment:
Oh yeah, my bad. WIll fix it soon. And I was still working on the docs... they just arrived